Services/IotHub/IotHubConnectionStringManager.cs

```diff
@@ -33,17 +33,21 @@ public class IotHubConnectionStringManager : IIotHubConnectionStringManager
         private readonly ILogger log;
         private readonly IDiagnosticsLogger diagnosticsLogger;
         private readonly IStorageRecords mainStorage;
-
+        private readonly IRegistryManager registryManager;
+        
```

Comment:
please remove spaces

Reply:
this is the same new line that was present between variables declaration and constructor. I did not add a new empty line

Reply:
there were 8 new spaces in the empty line, I see this has been fixed, thanks